cmd/immugw/immugw.go

```diff
@@ -29,6 +29,12 @@ import (
 	"github.com/spf13/viper"
 )
 
+var App = "immugw"
+var Version = "v0.0.1"
```

Comment:
Why we need to populate the var version? It should arrives in the compiling time. Shouldn't?